src/AuditableObserver.php

```diff
@@ -92,6 +92,10 @@ protected function dispatchAudit(Auditable $model): void
         if (! $model->readyForAuditing()) {
             return;
         }
+        $modelClass = $model::class;
+        if (method_exists($modelClass, 'isAuditingEnabled') && ! $modelClass::isAuditingEnabled()) {
```

Comment:
it seems like one check has been replaced by two, is this one definitely needed? should there be a separate test to pick it up?